Source/Editor/Content/Items/ContentItem.cs

```diff
@@ -706,25 +708,18 @@ public override void Draw()
                     var shadowRect = new Rectangle(2, 2, clientRect.Width + 1, clientRect.Height + 1);
                     var color = Color.Black.AlphaMultiplied(0.2f);
                     Render2D.FillRectangle(shadowRect, color);
-                    
-                    Render2D.FillRectangle(clientRect, style.Background.RGBMultiplied(1.25f));
-                    Render2D.FillRectangle(TextRectangle, style.LightBackground);
-                    
+
                     var accentHeight = 2 * view.ViewScale;
                     var barRect = new Rectangle(0, thumbnailRect.Height - accentHeight, clientRect.Width, accentHeight);
-                    Render2D.FillRectangle(barRect, Color.DimGray);
-                    
-                    DrawThumbnail(ref thumbnailRect, false);
+                            
                     if (isSelected)
-                    {
-                        Render2D.FillRectangle(textRect, Parent.ContainsFocus ? style.BackgroundSelected : style.LightBackground);
-                        Render2D.DrawRectangle(clientRect, Parent.ContainsFocus ? style.BackgroundSelected : style.LightBackground);
-                    }
+                        Render2D.FillRectangle(clientRect, Parent.ContainsFocus ? style.BackgroundSelected : style.LightBackground);
                     else if (IsMouseOver)
-                    {
-                        Render2D.FillRectangle(textRect, style.BackgroundHighlighted);
-                        Render2D.DrawRectangle(clientRect, style.BackgroundHighlighted);
-                    }
+                        Render2D.FillRectangle(clientRect, style.BackgroundHighlighted);
+                    else
+                        Render2D.FillRectangle(clientRect, style.LightBackground);
```

Comment:
The issue that I see with this is that the highlighting on some of the tiles looks different. What if we did what was there before with the "window" around the thumbnail but also add it to the folder item to keep it consistent?

Reply:
i just made consistent Highlights on gray scale offset, nothing more and remove some stuff with dont add to overall feel of the tile
and i doubt the [ContentItem.cs](https://github.com/FlaxEngine/FlaxEngine/pull/1378/files/85081a095d72f36a9bf4ac2e9a3bcb38df866b1a#diff-1885657851b3c22966481a12b5641354c604bf69618a98f118dc2bb758b895fb) is being not drawnd outside the bounds of scroll virew so less draw calls will be better